index.ts

```diff
@@ -10,30 +11,13 @@ export function convert(bin: string, theme?: string) {
             `./presets/${ theme || th || "default" }.html`,
             { encoding: "utf8" }
         ));
+		
         const { files } = await SourceBin.get(bin);
         const [ file ] = files;
-        const content = file.content
-            .replace(/^ {4}/gm, "  ")
-            .replace(/\r/g, '')
-            .split(/^/gms)
-            .map((val, i, array) => {
-                return `<line><num>${ i + 1 }</num><span>${
-                    val.replace(/</g, "&lt;")
-                        .replace(/</g, "&gt;")
-                }</span></line>`
-            })
-            .map((val, i, array) => {
-                const [ , maxLine ] = array[array.length - 1]
-                    .match(/<num>(\d+)<\/num>/) || [ undefined, '0' ]
-                const lineNumber = String(i + 1);
-                const line = lineNumber.padStart(maxLine.length, " ")
-                return val.replace(
-                    new RegExp(`<num>${ lineNumber }</num>`, "g"),
-                    `<num>${ line }</num>`
-                )
-            })
-            .join('')
-        const html = preset.replace(/{{ contents }}/, content)
+        const content = file.content;
+		
+const parsed = highlight(file.language.aceMode, content).value;
```

Comment:
```suggestion
        const parsed = highlight(file.language.aceMode, content).value;
```
Bruh I forgot to add the space